bfvcheck

```diff
@@ -27,6 +27,9 @@
 # of the authors and should not be interpreted as representing official policies,
 # either expressed or implied, of the FreeBSD Project.
 
+# inform that Linux is up through the rshim
+/usr/bin/bfrshlog "Linux up"
```

Comment:
There is a little concern that what if user calls the bfvcheck manually or multiple times... Maybe we should add it into bfvcheck.service instead of bfvcheck? We could add Spencer as reviewer as well.

Reply:
> There is a little concern that what if user calls the bfvcheck manually or multiple times... Maybe we should add it into bfvcheck.service instead of bfvcheck? We could add Spencer as reviewer as well.

For example:
ExecStartPre=/usr/bin/bfrshlog "Linux up"
ExecStart=/usr/bin/bfvcheck

Reply:
Yes will do